src/styles/index.scss

```diff
@@ -1,3 +1,133 @@
 body {
   margin: 0;
+
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  width: 200px;
+  height: 408px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  display: flex;
+  flex-direction: column;
+  padding-left: 16px;
+
+  &__photo {
+    width: 160px;
+    height: 134px;
+    background-size: contain;
+    background-repeat: no-repeat;
+    padding-top: 32px;
+    padding-left: 3px;
+  }
```

Comment:
You have padding applied to the 'card__photo' element. It's better to apply padding to the parent element ('card') to avoid repetitive padding declarations in child elements. This will make your code cleaner and easier to maintain.